src/SourceBuild/Arcade/tools/TextOnlyPackages.csproj

```diff
@@ -11,35 +11,10 @@
     should be added to source-build-reference-packages.
   -->
   <ItemGroup>
-    <PackageDownload Include="Microsoft.DotNet.Common.ItemTemplates"
```

Comment:
Are all of the PackageDownloads that are being removed not used in the Windows build?

Reply:
They are not required because source-build does not currently support Windows.

Reply:
I see, I missed that this was a source-build only project file.